pkg/dataplane/dataplane.go

```diff
@@ -88,6 +92,13 @@ func (dp *DataPlane) prepareEcho() {
 	apiv1.GET("/fact_types", dp.QueryFactTypes)
 }
 
+func (dp *DataPlane) prepareJSV1() {
+	jsv1 := dp.Echo.Group("/js/v1")
+	jsv1.Use(dp.middlewareAPIAudit())
+	jsv1.Use(dp.middlewareGrantValidation())
+	jsv1.POST("/facts", dp.CreateFact, dp.middlewareSetContext("scope_custom_id", ""))
```

Comment:
@jperichon the difference between `jsv1` and `apiv1` is that I override the scope_custom_id in the echo.Context, so that the handler `dp.CreateFact` will use the value `""` for scope_custom_id.